src/huggingface_hub/constants.py

```diff
@@ -245,3 +245,6 @@ def _as_int(value: Optional[str]) -> Optional[int]:
 HUGGINGFACE_HEADER_X_XET_EXPIRATION = "X-Xet-Token-Expiration"
 HUGGINGFACE_HEADER_X_XET_HASH = "X-Xet-Hash"
 HUGGINGFACE_HEADER_X_XET_REFRESH_ROUTE = "X-Xet-Refresh-Route"
+
+default_xet_cache_path = os.path.join(HF_HOME, "xet")
+HF_XET_CACHE = os.getenv("HF_XET_CACHE", default_xet_cache_path)
```

Comment:
`HF_XET_CACHE` should be added to the package reference https://github.com/huggingface/huggingface_hub/blob/06c4fbac44fb7f2b81e26ed17917e7a9258b1dff/docs/source/en/package_reference/environment_variables.md?plain=1#L32 (just after `HF_HUB_CACHE`)